### PR TITLE
Make level 2

### PR DESCRIPTION
**Pull request by Kondratovich**

### CODE REVIEW

File: DevIncubator-Autopark/CarColor.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal enum CarColor
+    {
+		Black,
```
Comment (kuzmiich) [inline DevIncubator-Autopark/CarColor.cs]: format spaces

File: DevIncubator-Autopark/Vehicle.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            VehicleType = vehicleType;
+            Model = model;
+            LicensePlate = licensePlate;
+            Weight = weight;
+            YearIssue = yearIssue;
+            Mileage = mileage;
+            Color = color;
+            TankCapacity = tankCapacity;
+        }
+        public double GetCalcTaxPerMonth() => (Weight * 0.0013) + (VehicleType.TaxCoefficient * 30) + 5;
```
Comment (kuzmiich) [inline DevIncubator-Autopark/Vehicle.cs]: For all numbers (if not -1, 0, 1) should create a constant variables

File: DevIncubator-Autopark/VehicleType.cs [hunk 3]
```diff
@@ -15,7 +9,7 @@ public VehicleType(string carType, double taxCoefficient = 1)
             TypeName = carType;
             TaxCoefficient = taxCoefficient;
         }
-        public override string ToString() => $"{TypeName},\"{TaxCoefficient}\"";
+        public override string ToString() => $"{TypeName};{TaxCoefficient}";
```
Comment (kuzmiich) [inline DevIncubator-Autopark/VehicleType.cs]:
```suggestion
        public override string ToString() => $"{TypeName},{TaxCoefficient}";
```

File: DevIncubator-Autopark/Vehicle.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        public override string ToString() => $"{VehicleType};{Model};{LicensePlate};{Weight};" +
```
Comment (kuzmiich) [inline DevIncubator-Autopark/Vehicle.cs]: replace `;` to `,`

File: DevIncubator-Autopark/Vehicle.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal class Vehicle : IComparable<Vehicle>
+    {
+        public readonly VehicleType VehicleType;
```
Comment (kuzmiich) [inline DevIncubator-Autopark/Vehicle.cs]: public field it`s bad practice.
Read information about autoproperty, or replace it on private fields with get and set(if field need it) methods

File: DevIncubator-Autopark/Vehicle.cs (hunk 2 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/Vehicle.cs]: fixed

File: DevIncubator-Autopark/VehicleType.cs (hunk 3 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/VehicleType.cs]: fixed

File: DevIncubator-Autopark/Vehicle.cs (hunk 4 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/Vehicle.cs]: fixed

File: DevIncubator-Autopark/Vehicle.cs (hunk 5 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/Vehicle.cs]: fixed

File: DevIncubator-Autopark/Vehicle.cs [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        }
+        public double GetCalcTaxPerMonth() => (Weight * WeightCoefficient) + (VehicleType.TaxCoefficient * TaxCoefficient) + TaxChange;
+        public override string ToString() => $"{VehicleType},{Model},{LicensePlate},{Weight}," +
+            $"{YearIssue},{Mileage},{Color},{TankCapacity},{GetCalcTaxPerMonth().ToString("0.00")}";
+        public int CompareTo(Vehicle other)
+        {
+            if (other is null)
+                throw new ArgumentNullException(nameof(other), "Error, argument cannot be null");
+            return GetCalcTaxPerMonth().CompareTo(other.GetCalcTaxPerMonth());
+        }
```
Comment (PushMeNow) [inline DevIncubator-Autopark/Vehicle.cs]: Could you place blank line between each of class member. For now it looks awful. I don't start and end of each of them.

File: DevIncubator-Autopark/VehicleHelper.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal static class VehicleHelper
+    {
+        public static void ShowAllVehicles(IReadOnlyList<Vehicle> vehicles)
```
Comment (PushMeNow) [inline DevIncubator-Autopark/VehicleHelper.cs]: Why is 'All' in the name of method? You could not know is it all or not all.

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: DevIncubator-Autopark/Vehicle.cs (hunk 6 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/Vehicle.cs]: fixed

File: DevIncubator-Autopark/VehicleHelper.cs (hunk 7 above)
Comment (Kondratovich) [reply DevIncubator-Autopark/VehicleHelper.cs]: fixed

**Review by PushMeNow — APPROVED** [review]

**Review by kuzmiich — APPROVED** [review]
Good job

**Pull request merged**